src/main/resources/META-INF/services/com.facebook.presto.spi.Plugin

```diff
@@ -0,0 +1 @@
+de.incub8.presto.uuid.UuidPlugin
```

Comment:
This file is automatically generated thanks to the `@MetaInfServices` annotation, remove it.